Wrappers/Python/ccpi/viewer/widgets/slider.py

```diff
@@ -0,0 +1,133 @@
+import vtk
+
+class SliderProperties:
+    def __init__(self):
+        self.tube_width = 0.004
+        self.slider_length = 0.025
+        self.slider_width = 0.015
+        self.end_cap_length = 0.008
+        self.end_cap_width = 0.02
+        self.title_height = 0.02
+        self.label_height = 0.02
+
+        self.value_minimum = 0.0
+        self.value_maximum = 1.0
+        self.value_initial = 1.0
+
+        self.orientation = 'horizontal'
+        self.offset = 0.12
+
+        self.p1 = [self.offset, self.end_cap_width * 1.1]
+        self.p2 = [1 - self.offset, self.end_cap_width * 1.1]
+
+        self.title = None
+
+        self.title_color = 'Black'
+        self.label_color = 'White'
+        self.value_color = 'White'
+        self.value_background_color = 'Black'
+        self.slider_color = 'Lime'
```

Comment:
- I think the colours Lime (and Yellow if kept) should be given as default parameters, the user can then decide the colour scheme. eg CIL could be magenta. 
- We could set all the colours to be default as this will make this class more customisable (just an idea).
- I think the yellow squares are redundant. They grey line already indicates the start and end of the range. It would be more appropriate to add the start and end points of the range in text (this would help, for example, the registration in IDVC, where the volume range is reduced). 
- The text "/max range value" is redundant if the max value is added as text. It could be simplified by having only the slice number sliding along with the green rectangle. 
- The green bar seems too large to indicate a single slice, I would make it thinner and taller.

Reply:
- The end squares are clickable and make the slider slide towards them
- I agree that the text Slice N/M over the slider would be redundant when we add minimum and maximum slice text at the ends of the slider bar. I tried and it is less trivial than it looks, so I would add it in a new issue
- I made the slider slightly smaller, but you need to be able to grab it and too small is too difficult